packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -5,10 +5,10 @@ import { join } from "path";
 
 const borderRadiusTransformations = [
   // tailwind classes
-  { from: /rounded(-[a-z]+)?-small/, to: "rounded$1-50" },
-  { from: /rounded(-[a-z]+)?-normal/, to: "rounded$1-100" },
-  { from: /rounded(-[a-z]+)?-large/, to: "rounded$1-150" },
-  { from: /rounded(-[a-z]+)?-circle/, to: "rounded$1-full" },
+  { from: /rounded(-[a-z]+)?-small\b/, to: "rounded$1-50" },
+  { from: /rounded(-[a-z]+)?-normal\b/, to: "rounded$1-100" },
+  { from: /rounded(-[a-z]+)?-large\b/, to: "rounded$1-150" },
```

Comment:
Works well, nice!

https://regex101.com/r/bnyJlh/1

I'm thinking that maybe we should add the word boundary at the beginning to be even safer. 

And also, WDYT about adding boundaries to other replacements, not only border-radius?

Reply:
> I'm thinking that maybe we should add the word boundary at the beginning to be even safer.

I thought that too, but I am concerned about things like `lm:rounded-small`? 🤔

Reply:
When I was "doing a research" I also got idea to set a boundary `\b` to the beginning, but then I had thoughts about "media queries" as Daniel mentions, so I added it only to the end. 🙈

Reply:
I also added the "end" boundaries to the other `from` regex expression within the codemod file (discussed with Daniel in PM).